test/serverTest.js

```diff
@@ -50,3 +53,153 @@ describe('server', () => {
       });
   });
 });
+
+describe('View Count', () => {
+  it('should update view count when progress equals one', (done) => {
+    const movieId = Math.floor(Math.random() * 300000);
+    const getViews = `select views from movies where id = ${movieId}`;
+    let viewsBefore;
+    let viewsAfter;
+    postgresDb.movieDb.query(getViews)
```

Comment:
You're using callbacks here a little bit like callbacks here. You can chain them so that when you get to an asynchronous function, the result gets passed to the next "then" in the chain. 
For example, instead of:

`asyncFn()`
    `.then((res1) => {`
        `// do stuff`
        `anotherAsyncFn()`
            `.then((res2) => {`
              `// do stuff`
            `});`
      `});`

You could try this (a little cleaner)

`asyncFn()`
    `.then((res1) => {`
        `// do stuff`
        `return anotherAsyncFn()`
      `})`
      `.then((res2) => {`
          `// do stuff`
      `});`